libnavigation-ui/src/main/java/com/mapbox/navigation/ui/VoiceInstructionCache.java

```diff
@@ -12,75 +12,73 @@
 
 class VoiceInstructionCache {
```

Comment:
hard to get what were changed. Could you revert to previous formatting?
Moreover, `CI` failed https://app.circleci.com/jobs/github/mapbox/mapbox-navigation-android/17755 on java-code checkstyle

Reply:
good catch! I think the checkstyle is different than my usual coding style regarding the indent. Just set up the Mapbox checkstyle, the code now should be clear.